integration/core/test_controller.py

```diff
@@ -163,18 +163,21 @@ def test_controller_expand(grpc_controller_client):  # NOQA
     assert v.replicaCount == 0
 
 
-def test_metric(grpc_controller_client):  # NOQA
-    replies = grpc_controller_client.metric_get()
-
+def test_metrics(grpc_controller_client):  # NOQA
```

Comment:
This test looks weird. Why there will be an exception probably happened? Also, why 5 iterations.

Reply:
I guess the purpose of this test is to check the values are always 0 if there is no IO. I feel it is not necessary.
I feel `data/test_basic_ops.py::test_rw_with_metric` is enough. I prefer to remove this test. WDYT?

Reply:
Agreed.

Reply:
Removed.